polkadot/node/network/statement-distribution/src/v2/mod.rs

```diff
@@ -754,22 +792,7 @@ fn find_active_validator_state(
 	let our_group = groups.by_validator_index(validator_index)?;
 
 	let core_index = group_rotation_info.core_for_group(our_group, availability_cores.len());
-	let paras_assigned_to_core = if let Some(claim_queue) = maybe_claim_queue {
-		claim_queue.iter_claims_for_core(&core_index).copied().collect()
-	} else {
-		availability_cores
```

Comment:
we're no longer using av-cores to get the scheduling state. Looks like the only place where we use the cores is to determine here the number of cores. This is equal to the number of validator groups which we already have on `Groups`

Reply:
what I meant is that now we can remove the av-cores runtime API call completely

Reply:
removed.